docs/en/seeding-classic-image.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Seeding a Classic Image"
+table_of_contents: true
+---
+
+# Seeding a Classic Image
+
+## Introduction
+A Classic image is seeded to include snaps and [assertions][assertions]. That
+is, you add the snaps and the assertions to the image so that on first boot it
+contains what your product needs. For example, a [gadget snap][gadget-snap] is
+normally seeded to point the system at a Serial Vault. Also, one seeds snaps
+to provide the software that customizes the system. This document explains
+what you may want to seed and how to do it.
+
+
+!!! Note:
+           Core images are seeded automatically with snaps and the model
+          assertion when created with the normal  ubuntu-image tool (available
```

Comment:
should be "`ubuntu-image`"?

Reply:
There are two spaces after "normal", get rid of one of them. :)